deploy/chart/catalog_resources/rh-operators/clusterloggings.crd.yaml

```diff
@@ -91,27 +101,46 @@ spec:
                     nodeSelector:
                       description: Define which Nodes the Pods are scheduled on.
                       type: object
-                    replicas:
+                    nodeCount:
                       description: Number of nodes to deploy for Elasticsearch
                       format: int32
                       type: integer
+                      minimum: 1
+                      default: 3
                     storage:
-                      description: 'The storage backing for Elasticsearch. More info: '
-                      type: object
+                      description: 'The storage specification for Elasticsearch data nodes'
+                      properties:
+                        size:
+                          description: 'The capacity of storage to provision.'
+                          type: string
+                          minimum: "10Gi"
+                          default: "200Gi"
+                        storageClassName:
+                          description: 'The class of storage to provision. More info: https://kubernetes.io/docs/concepts/storage/storage-classes/'
+                          type: string
```

Comment:
default value of `"gp2"` ?

Reply:
Architecture team stated that "gp2" is the default if you dont define one on AWS

Reply:
so should this default to `""` then? Or should the elasticsearch operator default it to `""` if its nil?

Reply:
defaults not supported by validation logic